charts/tfy-inferentia-operator/templates/neuron-scheduler.yaml

```diff
@@ -122,6 +122,10 @@ spec:
         version: second
     spec:
       priorityClassName: system-cluster-critical
+      {{- with .Values.scheduler.imagePullSecrets | default .Values.imagePullSecrets }}
```

Comment:
there is no reference to .Values.scheduler.imagePullSecrets in the values.yaml file. It can be an empty list mainly for reference purpose

Reply:
have made the changes, added those as well.

Reply:
affinity docs were missing, have added those as well.